src/components/action-button.tsx

```diff
@@ -114,7 +114,7 @@ export const ActionButton = ( {
 			};
 			break;
 		case 'loading':
-			buttonLabel = __( 'Starting…' );
+			buttonLabel = __( 'Loading…' );
```

Comment:
What do you think about displaying different labels?

```suggestion
			buttonLabel = isRunning ? __( 'Stopping…' ) : __( 'Starting…' );
```


https://github.com/user-attachments/assets/e395b74d-567f-464b-8348-77a8e47a3473

Reply:
Thanks, makes sense. Changed: https://github.com/Automattic/studio/pull/581/commits/5d72de829b41d7b976fa5cdea4408f09dfaf098b